data/txntest/txn.go

```diff
@@ -108,10 +109,20 @@ type Txn struct {
 
 // Noted returns a new Txn with the given note field.
 func (tx *Txn) Noted(note string) *Txn {
-	copy := &Txn{}
-	*copy = *tx
+	copy := *tx
```

Comment:
nit: you could declare this function on `Txn` instead of `*Txn` to avoid having to create a copy. Same thing for `Args` below